### PR TITLE
added option for skipping git initialization

### LINKED ISSUE #12: Add a way to skip git
Some users might not use git.  It'd be nice to offer a `--skip-git` option we could pass to the boilerplate during setup.

### PR DESCRIPTION
**Pull request by mwarger**
resolved #12
todo: document availability in ignite

### CODE REVIEW

File: boilerplate.js [hunk 1]
```diff
@@ -21,19 +21,21 @@ const finish = async function (context) {
...
-    await system.run('git init .')
-    ignite.log('git add .')
-    await system.run('git add .')
-    ignite.log('git commit')
-    await system.run('git commit -m "Initial commit."')
-    // setup husky git hooks
-    spinner.text = 'setting up git hooks'
-    system.run(`node node_modules/husky/bin/install .`)
-    spinner.succeed('configured git')
+  if (!parameters.options['skip-git']) {
```
Comment (skellock) [inline boilerplate.js]: Unfortunately, due to the way `minimist` works, we have to explicitly check for `=== false` and not falsy. It feels wierd.

**Review by skellock — CHANGES_REQUESTED** [review]
Just need to touch up that expression to check for `=== false`.

File: boilerplate.js (hunk 1 above)
Comment (mwarger) [reply boilerplate.js]: I see.  Thanks.

File: boilerplate.js [hunk 2]
```diff
@@ -21,19 +21,21 @@ const finish = async function (context) {
...
-    await system.run('git init .')
-    ignite.log('git add .')
-    await system.run('git add .')
-    ignite.log('git commit')
-    await system.run('git commit -m "Initial commit."')
-    // setup husky git hooks
-    spinner.text = 'setting up git hooks'
-    system.run(`node node_modules/husky/bin/install .`)
-    spinner.succeed('configured git')
+  if (parameters.options['skip-git'] === false) {
```
Comment (skellock) [inline boilerplate.js]: I'm sorry.  I was wrong.  It's `!== true`.   

When you set --skip-git, it shows up in options as `{ 'skip-git': true }`.  When you omit it, options is `{}` and if you type --no-skip-git it becomes `{'skip-git': false}`.

I'm sorry for that.

Give it a test with & without and make sure it's doing the right thing.

Oh boy, it's late.  I'm not thinking clearly.  :)

**Comment by skellock** [comment]
I'm dumb.  Sorry @mwarger .

**Comment by mwarger** [comment]
Haha, no problem.  It's late for me too, I will finish this up tomorrow.

**Comment by mwarger** [comment]
Should be good now, I think.  Tested with `--skip-git`, `--no-skip-git`, and by omission.

**Review by skellock — APPROVED** [review]

**Comment by skellock** [comment]
Thank you. 😊

**Pull request merged**